x-pack/legacy/plugins/siem/server/lib/framework/kibana_framework_adapter.ts

```diff
@@ -89,35 +96,6 @@ export class KibanaBackendFrameworkAdapter implements FrameworkAdapter {
         }
       }
     );
-
-    if (!this.isProductionMode) {
```

Comment:
there is no separate endpoint for `graphiql` anymore

Reply:
@patrykkopycinski does that mean that this functionality exists within graphql now, or that we're losing this functionality entirely? /cc @XavierM since I know he was most familiar with usage of this endpoint.

Reply:
it's available under the graphql endpoint 
https://www.apollographql.com/docs/apollo-server/testing/graphql-playground/